.github/workflows/ci.yml

```diff
@@ -16,6 +16,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@1.88.0
+
+      - name: Install Rust Fmt and Clippy
+        shell: bash
+        run: rustup component add rustfmt clippy
```

Comment:
If we want to also enforce this outside the CI, we could use the `rust-toolchain.toml` file. If I understand the [documentation](https://rust-lang.github.io/rustup/overrides.html) correctly, `rustup` should then automatically install / pick the specified version when commands are issued.

Reply:
I think it's just a file at the repo level and rustup should use it automatically. I've never used it myself so far though. Not a real issue. Was just a thought.

Reply:
I think @Johennes is right; if we don't want to do that, then we can [put it in the manifest](https://doc.rust-lang.org/cargo/reference/rust-version.html).